content/source/docs/enterprise/admin/active-active.html.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: "enterprise"
+page_title: "Active/Active - Infrastructure Administration - Terraform Enterprise"
+---
+
+# Terraform Enterprise Active/Active
+
+When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](https://www.terraform.io/docs/enterprise/install/active-active.html) for more information on moving to Active/Active.
+
+## Active/Active Admin Commands
+
+The Active/Active operational mode disables use of the Replicated Admin Console.  To accommodate this, there are now admin commands to facilitate configuration changes, safe application stops, and producing support bundles. There is a new container in the TFE architecture to support this called "tfe-admin". You must login to a node in the Active/Active cluster using SSH to run these commands from the command line.
```

Comment:
This might be a good place to explain that the commands should be run on the target node itself, or should we state that in the description of each command. Thoughts?

Reply:
updated in commands based on other feedback also